suite-native/atoms/src/Select/Select.tsx

```diff
@@ -23,36 +28,68 @@ export const Select = <TItemValue extends SelectItemValue>({
     selectLabel,
     selectValue,
     onSelectItem,
+    isConfirmable = false,
     testID,
 }: SelectProps<TItemValue>) => {
     const { bottomSheetRef, openModal, closeModal } = useBottomSheetModal();
 
-    const selectedItem = useMemo(
-        () => items.find(item => item.value === selectValue),
-        [selectValue, items],
+    const selectTriggerValue = useMemo(
+        () => items.find(item => item.value === selectValue)?.label ?? null,
+        [items, selectValue],
     );
-    const handleSelectItem = (itemValue: TItemValue) => {
+
+    const [selectedItemValue, setSelectedItemValue] = useState(selectValue);
+    const [isConfirmButtonVisible, setIsConfirmButtonVisible] = useState(false);
+
+    const openBottomSheet = () => {
+        setSelectedItemValue(selectValue);
```

Comment:
nit: To make it clearer I would personally change this line to:
```suggestion
        if(isConfirmable){
            setSelectedItem(defaultItem);
        }       
```